src/containers/FormContainer/FormContainer.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import './FormContainer.css';
+import HeaderForm from '../../components/Forms/HeaderForm/HeaderForm';
+import ListForm from '../../components/Forms/ListForm/ListForm';
+import FooterForm from '../../components/Forms/FooterForm/FooterForm';
+import CompletedTodos from '../../components/Forms/CompletedTodos/CompletedTodos';
+import NotCompletedTodos from '../../components/Forms/NotCompletedTodos/NotCompletedTodos';
+import {Route} from 'react-router-dom';
+
+const FormContainer = (props) => {
+
+
+    return (
+        <div className="form-container">
+
+            {/* #############      HEADER     ############ */}
+            <Route path="/"
+                   render={() => (
+                       <HeaderForm
+                           postTodoHandler={props.postTodoHandler}
+                           fetchDataHandler={props.fetchDataHandler}
+                       />
+                   )}
+            />
+
+
+            <Route path="/" exact
+                   render={() => (
+                       <ListForm
+                           todos={props.todos}
+                           deleteTodoHandler={props.deleteTodoHandler}
+                           putTodoContentHandler={props.putTodoContentHandler}
+                           putTodoStatusHandler={props.putTodoStatusHandler}
+
+                       />)
+                   }
+            />
+
+
+            <Route path="/completed-todos" exact
+                   render={() => (
+                       <CompletedTodos
+                           completedTodos={props.completedTodos}
+                           deleteTodoHandler={props.deleteTodoHandler}
+                           putTodoContentHandler={props.putTodoContentHandler}
+                           putTodoStatusHandler={props.putTodoStatusHandler}
+
+                           notCompletedTodosNum={props.notCompletedTodosNum}
+                       />
+                   )}
+            />
+
+
+            <Route path="/not-completed-todos" exact
+                   render={() => (
+                       <NotCompletedTodos
```

Comment:
Whats the difference between `NotCompletedTodos` and `CompletedTodos`.
I feel like it can be on component with different props.

Reply:
Actually the different was just if the todo status is true or false. 
I have noticed that it is just a small difference but, I wanted to have 2 separate components for using react router. 
What I wanted to do was having two Link for completed todo, and not-completed todo . And I thought making two components which has a slight difference for that purpose.

Reply:
> I have noticed that it is just a small difference but, I wanted to have 2 separate components for using react router.
You can use the same Component for different routes.

> What I wanted to do was having two Link for completed todo, and not-completed todo . And I thought making two components which has a slight difference for that purpose.

If that is so, you should create one generic `Todos` component which will be wrapped by `CompletedTodos` and `NotCompletedTodos`.
try to make your code DRY (don't repeat yourself) as possible :+1: